direct/core/src/test/java/cz/o2/proxima/direct/storage/InMemStorage.java

```diff
@@ -307,7 +308,12 @@ public ObserveHandle observePartitions(
         Position position,
         boolean stopAtCurrent,
         LogObserver observer) {
-      return observe(null, position, stopAtCurrent, observer);
+      return observe(
+          null,
+          position,
+          partitions.stream().map(ConsumedOffset::empty).collect(Collectors.toList()),
+          stopAtCurrent,
+          observer);
```

Comment:
Why this change? It should work as before, right? The call to `observe(null, position, stopAtCurrent, observer)` is public and thus should work.

Reply:
Because we need to take "partitions" parameter into account? (previous implementation contains bug and was not cover with tests)

Reply:
Ah, yep. :)